developer-access.tf

```diff
@@ -0,0 +1,30 @@
+data "azuread_group" "developers" {
+  name = var.developers_group
+}
+
+locals {
+  is_prod = length(regexall(".*(prod).*", var.env)) > 0
```

Comment:
Are we likely to use this module with env names like `predprod`, `nonprod`, etc?

Reply:
if we are then we need to adjust the check, cnp-module-postgres also has this check too